test/tracking.js

```diff
@@ -0,0 +1,180 @@
+const chai = require('chai');
+const sinon = require('sinon');
+const sleep = require('sleep-promise');
+
+const errors = require('../api/v1/errors');
+const config = require('../api/config');
+const TrackingItem = require('../api/v1/models/TrackingEvent');
+const TrackingService = require('../api/v1/services/TrackingService');
+
+const assert = chai.assert;
+const expect = chai.expect;
+const tracker = require('mock-knex').getTracker();
+const fakeredis = require('fakeredis');
+
+const _REDIS_CONFIG = {
```

Comment:
We shouldn't need to do this, with the old system the cache instance was mocked when the `env` was test, otherwise it connected to the specified cache